pootle/apps/pootle_project/models.py

```diff
@@ -224,7 +225,11 @@ def accessible_by_user(cls, user):
 
         :param user: The ``User`` instance to get accessible projects for.
         """
-        username = 'nobody' if user.is_anonymous() else user.username
+        username = user.username
+        if user.is_anonymous():
+            username = 'nobody'
+        elif user.is_superuser:
+            username = 'admin'
```

Comment:
I'm not sure I get the logic behind this. Is it to (micro)optimize the number of cached entries for administrators?
Note besides the fact that Pootle ships with an `admin` superuser, `admin` is not a special or reserved username at all, so in a scenario where a regular user named `admin` exists, it could entirely bypass any permission-checking, or disallow admins from having full access (depending on who populated the cache first).